src/main/java/com/google/sps/servlets/utils/resources/common_words.txt

```diff
@@ -0,0 +1,22 @@
+the
+it
+at
+and
+to
+we
+can
+are
+of
+its
+it
+am
+my
+or
+by
+for
+i
+my
+this
+was
+is
+a
```

Comment:
Can we make them in alphabetical order like a dictionary? I am getting to used to sorted lists oops...